src/content/docs/Company/security.md

```diff
@@ -0,0 +1,186 @@
+Devices owned and issued by the company and are required to use full disk
+encryption.
+
+We do not permit access to Shorebird production systems or customer data from
+personal devices.
+
+The vast majority of Shorebird's code and operations are handled in the open,
+either via public code on GitHub, or public discussions on Discord, so while we
+do not permit access to internal systems from personal devices, employees
+generally do not need access to internal systems to do their work.
+
+### Passwords
+
+All passwords are stored in a password manager. We use
+[1Password](https://1password.com/). We do not store passwords in code, in
+emails, or in any other insecure location. Passwords should never be passed to a
+command line program as an argument, as they will then be stored in the shell
+history file.
+
+We do not recommend ever typing a password. Instead, we recommend using a
+password manager to generate and store passwords. This ensures that passwords
+will never be input into a place they were not intended to be.
+
+Access to our SSO provider (Google) has required strong passwords (Google's
+guidelines) and two factor authentication (2FA).
+
+### Physical Access
+
+Shorebird is an
+[all-remote company](/company/company_details/#a-note-on-remote-only). We have
+no physical office or visitor policies. We do not own or operate physical
+servers.
+
+### User Access Review
+
+We review all user access to our systems periodically, as well as as part of an
+employee joining or leaving the company. All access to Shorebird systems is
+gated through Google SSO including required two factor authentication.
+
+## Third Party Suppliers
+
+We use a number of third party services to run our business. We list those which
+may come in contact with customer data as part of our privacy policy:
+https://shorebird.dev/privacy
+
+### Third Party Service Accounts
+
+Accounts are bucketed into having handling customer data (e.g. Google Cloud) or
+not (e.g. Canva).
+
+- All accounts and services we use are logged in our Accounts sheet.
+- All accounts should authenticate through Google OAuth (SSO). Exceptions must
+  be approved by the CEO.
+- Accounts that can never reach customer data (e.g. Twitter), may alternatively
+  use a strong password stored in 1Password and two factor authentication if SSO
+  is not available.
+- Additions to the Accounts sheet are reviewed by the CEO.
+
+## Incident Response
+
+We have a private playbook for incident response. We have logging and alerting
+in place to detect and respond to incidents. We have both dedicated private
+channels on Discord for response as well as back-up text communication pathways
+as well as phone numbers for all engineers.
+
+We do not currently have separate incident tracking beyond our public GitHub. We
+always notified all customers when affected by incidents (security or otherwise)
+via their billing email address in the past and will continue to do so going
+forward.
+
+## Vendor Certifications
```

Comment:
We should just change this to say ISO 27001 is in progress, expected completion mid 2025.